src/Compilers/Core/Portable/Operations/OperationInterfaces.xml

```diff
@@ -831,7 +831,7 @@
       </Comments>
     </Property>
   </Node>
-  <Node Name="IUnaryOperation" Base="IOperation">
+  <Node Name="IUnaryOperation" Base="IOperation" SkipInVisitor="true">
```

Comment:
why?

Reply:
is it because otehr ops inherit from it?  If so, my prefernece would instead be that this skipping should be computed based on the already expressed relationship between operations.

Reply:
No, `Node` is a leaf. It's because the name was inconsistent. I changed the approach in the next commit to actually record the inconsistent name, as it's needed to generate the `Accept` method.